CHANGES.md

```diff
@@ -1,5 +1,23 @@
 # Changes
 
+## edge-21.7.4
+
+This release continues to focus on dependency updates. It also adds proxy a
+change that add a new `l5d-proxy-error` information header to distinguish
+proxy generated errors from application generated errors. It also adds
+support for proxies in gateway mode to now have `endpoints` as gateway targets.
+
+* Updated several project dependencies
+* Added support for gateway proxies to use `endpoint` targets in addition to
```

Comment:
let's omit this for now -- it's only user-facing when Matei's PR lands to support statefulsets
```suggestion
```